[12.0][ADD] New delivery note modules (#1840)
OCA/l10n-italy: l10n_it_delivery_note_order_link/readme/USAGE.rst

**English**

> To use this module, you need to:
> 
> * from the purchase order a smart button link to delivery notes.

Reviewer: È rimasta aperta anche [questa](https://github.com/OCA/l10n-italy/pull/1840#discussion_r495788918).

Un suggerimento per agevolarti:

```suggestion
A smart button will appear on each purchase order, so you can link them to DNs.
```

**English**

To use this module, you need to:

* from the purchase order a smart button link to delivery notes.

**Italiano**

Per usere questo modulo hai bisogno di :

> * su ordini di acquisto c'è uno smart button che collega ai DDT.

Reviewer: > In caso andrà modificata anche la stringa in inglese.

Anche qui andrebbe uniformata per essere in linea con quella in italiano.
